src/languages/swift.js

```diff
@@ -491,14 +490,56 @@ export default function(hljs) {
     ]
   };
 
+  function noneOf(list) {
+    return negativeLookahead(either(...list));
+  }
+
+  const METHODS_ONLY = [...Swift.keywords, ...Swift.builtIns];
+  const FUNCTION_CALL = {
+    relevance: 0,
+    variants: [
+      {
+        // Functions and macro calls
+        scope: "title.function",
+        keywords: KEYWORDS,
+        match: concat(
+          either(/\b/, /#/),
+          noneOf(METHODS_ONLY.map(x => concat(x, TRAILING_PAREN_REGEX))),
+          FUNCTION_IDENT,
+          lookahead(TRAILING_PAREN_REGEX),
+        ),
+      },
+      {
+        // Keywords/built-ins that only can appear as a method call
+        // e.g. foo.if()
+        match: [
+          /\./,
+          either(...METHODS_ONLY),
+          TRAILING_PAREN_REGEX,
+        ],
+        scope: {
+          2: "title.function",
```

Comment:
We'd call anything that *appears* to be an object property access a `property` for consistent highlighting across different grammars.